### PR TITLE
Correct link to all changes in 1.15

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: _includes/manuals/1.15/1.2_release_notes.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ A long-wanted feature, now it is possible to just add a connection to a Compute
 #### Various fixes and features
 * Fix 414 when manipulating entities with long name ([#17850](http://projects.theforeman.org/issues/17850))
 
-*A full list of changes in 1.15.0 is available via [Redmine](http://projects.theforeman.org/projects/foreman/issues?query_id=126)*
+*A full list of changes in 1.15.0 is available via [Redmine](http://projects.theforeman.org/projects/foreman/issues?release_id=209)*
```
Comment (ohadlevy) [inline _includes/manuals/1.15/1.2_release_notes.md]: this url lists all the open issues left in 1.15...

**Review by ohadlevy — CHANGES_REQUESTED** [review]

File: _includes/manuals/1.15/1.2_release_notes.md (hunk 1 above)
Comment (tbrisker) [reply _includes/manuals/1.15/1.2_release_notes.md]: are you sure? I don't see any open issues there

Comment (ohadlevy) [reply _includes/manuals/1.15/1.2_release_notes.md]: @tbrisker I'm sure, there are some cookies in use...

**Comment by tbrisker** [comment]
Updated, hopefully got the correct link this time.

**Comment by mmoll** [comment]
merged, thanks @tbrisker!

**Pull request merged**